MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/layers/CustomLayer.java

```diff
@@ -20,16 +20,9 @@ public CustomLayer(String id,
     super(nativePtr);
   }
 
-  public void update() {
```

Comment:
Maybe it would be better to deprecate it https://github.com/mapbox/mapbox-gl-native-android/commit/939f9518ab31da33feda5e4a89d3b30c0e0590c3